### PR TITLE
 basic support for .netstandard libraries

### PR DESCRIPTION
**Pull request by DevNamedZed**
add new make target 'netstandrd' for building .net standard assembly that is compatible with .net core

### CODE REVIEW

File: makefiles/Makefile.csharp.mk [hunk 1]
```diff
@@ -145,7 +177,7 @@ $(GEN_DIR)/com/google/ortools/properties/AssemblyInfo.cs: \
 	$(COPY) tools$Scsharp$SAssemblyInfo.cs $(GEN_DIR)$Scom$Sgoogle$Sortools$Sproperties
 ifdef CLR_KEYFILE
 ifeq ($(SYSTEM),win)
-	@echo [assembly: AssemblyKeyFile("$(CLR_KEYFILE)")] >> $(GEN_DIR)$Scom$Sgoogle$Sortools$Sproperties$SAssemblyInfo.cs
+	@echo [assembly: AssemblyKeyFile("$(CLR_KEYFILE))")] >> $(GEN_DIR)$Scom$Sgoogle$Sortools$Sproperties$SAssemblyInfo.cs
```
Comment (lperron) [inline makefiles/Makefile.csharp.mk]: bizarre extra )

**Comment by Mizux** [comment]
Travis fails due to some update in Cbc cmake support, this is not related to this patch (which is not tested by CI)

**Comment by lperron** [comment]
Hi,

Could you document a bit how to use it?

Laurent Perron | Operations Research | lperron@google.com | (33) 1 42 68 53
00


2018-02-06 16:54 GMT+01:00 Mizux <notifications@github.com>:

**Comment by DevNamedZed** [comment]
Hi guys,

Thanks for review and getting checked in. I will submit another PR for code signing, version, and documentation later on today.

Thanks
Ziad

**Pull request merged**